enterprise/utils.py

```diff
@@ -519,6 +534,13 @@ def enterprise_enrollment_source_model():
     return apps.get_model('enterprise', 'EnterpriseEnrollmentSource')  # pylint: disable=invalid-name
 
 
+def enroll_notification_email_template():
+    """
+    Returns the ``EnrollmentNotificationEmailTemplate`` class.
+    """
+    return apps.get_model('enterprise', 'EnrollmentNotificationEmailTemplate')  # pylint: disable=invalid-name
```

Comment:
what's the reason behind a separate method that just fetches a model?

Reply:
guess I just blindly followed the examples I saw for the other models. I changed it to call `apps.get_model('enterprise', 'EnrollmentNotificationEmailTemplate')` directly for now as esp. there is only one call site for it anyway